addons/binding/org.openhab.binding.exec/README.md

```diff
@@ -38,27 +41,121 @@ The following parameters are automatically added:
 All Things support the following channels:
 
 | Channel Type ID | Item Type    | Description  |
-|-----------------|------------------------|--------------|----------------- |------------- |
+|-----------------|------------------------|--------------|
 | input | String       | Input parameter to provide to the command |
 | output | String       | Output of the last execution of the command |
 | exit | Number       | The exit value of the last execution of the command |
 | run | Switch       | Send ON to execute the command and the current state tells whether it is running or not |
 | lastexecution | DateTime       | Time/Date the command was last executed, in yyyy-MM-dd'T'HH:mm:ss.SSSZ format |
 
-## Full Example
+## Short Example
 
 **demo.things**
 
-```
-Thing exec:command:apc [command="/usr/local/bin/apcaccess  status", interval=15, timeout=5]
+```java
+Thing exec:command:apc [command="/usr/local/bin/apcaccess status", interval=15, timeout=5]
 Thing exec:command:myscript [command="php ./configurations/scripts/script.php %2$s", transform="REGEX((.*?))"]
 ```
 
 **demo.items**
 
-```
-String APCRaw "[%s]" (All) {channel="exec:command:apc:output"} 
-Switch APCRunning { channel="exec:command:apc:run"}
+```java
+String APCRaw "[%s]" (All) {channel="exec:command:apc:output"}
+Switch APCRunning {channel="exec:command:apc:run"}
 Number APCExitValue {channel="exec:command:apc:exit"}
 DateTime APCLastExecution {channel="exec:command:apc:lastexecution"}
 ```
+
+## Full Example
+
+Following is a example shown how to read out the temperature of the RPI with all files needed to set up and debug or log the executed command.
```

Comment:
Please change to Raspberry Pi (the abbreviation should be RPi) and expect the reader to not know it -> "of the single board computer "Raspberry PI""